src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/snippet/Style.java

```diff
@@ -37,21 +37,21 @@
 
     /**
      * A style that describes a link. Characters of this style are typically
-     * processed by wrapping into an HTML {@code A} element pointing to the
+     * processed by being wrapped into an HTML {@code A} element pointing to the
      * provided target.
      */
     record Link(String target) implements Style { }
 
     /**
      * A named style. Characters of this style are typically processed by
-     * wrapping into an HTML {@code SPAN} element with the {@code class}
+     * being wrapped into an HTML {@code SPAN} element with the {@code class}
      * attribute which is obtained from the provided name.
      */
     record Name(String name) implements Style { }
 
     /**
      * A marker of belonging to markup. Characters of this style are typically
```

Comment:
The initial sentence is a little awkward. Can it be reposed to be clearer?

Reply:
Feel free to take a stab at it; I'm not good with words.